ceno_zkvm/src/scheme/verifier.rs

```diff
@@ -117,7 +117,7 @@ impl<E: ExtensionField, PCS: PolynomialCommitmentScheme<E>> ZKVMVerifier<E, PCS>
             }
         }
 
-        for (name, (_, proof)) in vm_proof.opcode_proofs.iter() {
+        for (name, (i, proof)) in vm_proof.opcode_proofs.iter() {
```

Comment:
Why the change here?  You don't use the variable `i`, do you?

Reply:
done